Packs/AWS-SNS/Integrations/AWSSNS/README.md

```diff
@@ -0,0 +1,242 @@
+Amazon Web Services Simple Notification Service (SNS)
+This integration was integrated and tested with version xx of AWS - SNS
```

Comment:
please specify the right version on which this was tested not `xx`

Reply:
@ilappe Fixed to 1.0.0